### ISSUE TITLE
WebExtension version

### ISSUE THREAD

**Issue by mems** [Opquast/opquast-desktop#21]
Firefox v57 (november 2017) will only support WebExtensions.

This rework could benefit to Chrome and Opera users that allow them to use Opquast Desktop on their browsers.

See [Porting a legacy Firefox extension - Mozilla | MDN](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Porting_a_legacy_Firefox_add-on) and [Extending the developer tools - Mozilla | MDN](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Extending_the_developer_tools)

**Comment by laurentj** [comment]
Yes we know. We already studied the case. [I even already started a new prototype](https://github.com/laurentj/opquast-desktop/tree/webext), but this migration is lot of works. And not enough time to continue. Not sure there will be a WebExtensions version of Opquast Desktop when Fx 57 will be released...

**Comment by fabolousfrances** [comment]
Any news or update ? FF57 is now released
OPQuast is now obsolet :(
Thanks

**Comment by mems** [comment]
I think it could be better if it's included it as an audit in [Lighthouse](https://github.com/GoogleChrome/lighthouse/blob/master/docs/new-audits.md) or as rule in [sonar](https://sonarwhal.com/docs/developer-guide/)

What do you think?
